server/models/account/user.ts

```diff
@@ -87,7 +104,7 @@ enum ScopeNames {
   ]
 }))
 @Scopes(() => ({
-  [ScopeNames.FOR_ME_API]: {
+  [ScopeNames.WITH_VIDEOCHANNELS]: {
```

Comment:
You still need a `ScopeNames.FOR_ME_API` scope that includes `UserNotificationSettingModel`